src/templates/project.js

```diff
@@ -122,7 +122,7 @@ const ProjectTemplate = ({ data, pathContext }) => {
         description={project.description}
       />
       <LeftPane>
-        {project.video && <Video video={project.video} />}
+        {project.video.vimeoId && <Video video={project.video} />}
```

Comment:
I think it might be safer to have `project.video && project.video.viemeoId` in case the video is null.

Reply:
Good call - I was assuming that the empty object default for `video` would take care of this but it is smart to check anyways in case the semantics change.